src/Features/CSharp/Portable/ConvertToRecord/ConvertToRecordEngine.cs

```diff
@@ -394,6 +446,75 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
             }));
         }
 
+        private static async Task RefactorInitializersAsync(
+            INamedTypeSymbol type,
+            SolutionEditor solutionEditor,
+            ImmutableArray<IPropertySymbol> positionalParameters,
+            CancellationToken cancellationToken)
+        {
+            var symbolReferences = await SymbolFinder
+                .FindReferencesAsync(type, solutionEditor.OriginalSolution, cancellationToken).ConfigureAwait(false);
+            var referenceLocations = symbolReferences.SelectMany(reference => reference.Locations);
+            var documentLookup = referenceLocations.ToLookup(refLoc => refLoc.Document.Id);
+            foreach (var (documentID, documentLocations) in documentLookup)
+            {
+                var documentEditor = await solutionEditor
+                        .GetDocumentEditorAsync(documentID, cancellationToken).ConfigureAwait(false);
+                if (documentEditor.OriginalDocument.Project.Language != LanguageNames.CSharp)
+                {
+                    // since this is a CSharp-dependent file, we need to have specific VB support.
+                    // for now skip VB usages.
```

Comment:
Let's make sure we have a tracking bug to link here. I think it makes sense to merge this before this is done so we can start getting dogfood coverage (since the number of folks doing cross-language is small enough), but we don't want to lose this.

Reply:
Tracked in #63756